src/script/mls/mlsConversationState/mlsConversationState.ts

```diff
@@ -55,7 +57,10 @@ export const useMLSConversationState = create<StoreState>((set, get) => {
   return {
     established: initialState.established,
     filterEstablishedConversations: conversations =>
-      conversations.filter(conversation => !conversation.groupId || get().isEstablished(conversation.groupId)),
+      conversations.filter(conversation => {
+        const isMLS = isMLSConversation(conversation);
+        return !isMLS || get().isEstablished(conversation.groupId);
+      }),
```

Comment:
After mixed conversations (that also contain `groupId` field) were introduced, we want to hide only `"mls"` conversations that are not established. We're totally fine with showing `"mixed"` conversations that don't have MLS group established - when conversation is `"mixed"`, we still send messages via Proteus.